packages/calcite-components/src/components/autocomplete-item/autocomplete-item.scss

```diff
@@ -76,7 +78,7 @@
 
 .container--active {
   @apply focus-inset;
-  background-color: var(--calcite-autocomplete-background-color, var(--calcite-color-foreground-2));
+  background-color: var(--calcite-autocomplete-background-color-focus, var(--calcite-color-foreground-2));
```

Comment:
I just realized `--calcite-autocomplete-background-color-hover` and `--calcite-autocomplete-background-color-focus` have the same default so they can share a token. Simplify this to `--calcite-autocomplete-background-color-hover` per the wiki guidelines

Reply:
updated

Reply:
doesn't look like this got updated